README.md

```diff
@@ -129,5 +129,6 @@ jobs.
 
 ## License
 
-Delayed::Cron::Job is released under the terms of the MIT License.
-Copyright 2014-2019 Pascal Zumkehr. See LICENSE for further information.
+Delayed::Cron::Job is released under the terms of the [MIT License](LICENSE).
+Copyright 2014-2019 Pascal Zumkehr. See see [LICENSE file](LICENSE) for further
+information.
```

Comment:
I think a single link is enough. Either keep the first occurrence (then I would drop the second sentence entirely), or the second one, where I would strip the "file" again as well as the doubled "see".